app.js

```diff
@@ -95,34 +92,27 @@ exports.init = function(isA11yTest = false, a11yTestSession = {}, ftValue) {
                 '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\'',
                 'www.google-analytics.com',
                 'www.googletagmanager.com',
-                'vcc-eu4.8x8.com',
-                'vcc-eu4b.8x8.com',
                 `'nonce-${nonce}'`,
-                'webchat-client.ctsc.hmcts.net',
+                'webchat.training.ctsc.hmcts.net',
```

Comment:
is it okay to remove the webchat-client.xxx url and only have training ? should we not have both ?

Reply:
yes we no longer calling webchat-training.ctsc.hmcts.net but we only use webchat.ctsc.hmcts.net